scripts/linux/compile.sh

```diff
@@ -38,13 +38,13 @@ fi
 if [ -d $parent_dir/output ]; then
 	read -p "Directory $parent_dir/output already exists. The script will delete and recreate it. Is that okay? (y/N) : " yn
 	case $yn in
-	    [Yy]* ) rm -rf $parent_dir/output ;;
+	    [Yy]* ) echo 1 ;;
 	    * ) echo "Aborted."; exit;;
 	esac
 fi
-
+#rm -rf $parent_dir/output
```

Comment:
Also a remaining of some work, that should not end up in the PR?

Reply:
yes will fix